### ISSUE TITLE
help -> report bug is broken

### ISSUE THREAD

**Issue by BMaxV** [PintaProject/Pinta#921]
**Description**
The help -> report bug function causes a "unhandled exception" pop up.

![image](https://github.com/PintaProject/Pinta/assets/5713211/e7dd5877-29f4-4f5d-ae8b-bea4ea8a56a3)

**To Reproduce**
I don't know, click the button?

Here are the details from the "unhandled exception" window:
```
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.ComponentModel.Win32Exception: Cannot find the specified file
  at System.Diagnostics.Process.StartWithShellExecuteEx (System.Diagnostics.ProcessStartInfo startInfo) [0x00129] in <a85c1a570f9a4f9f9c3d2cfa5504e34f>:0 
  at System.Diagnostics.Process.Start () [0x00038] in <a85c1a570f9a4f9f9c3d2cfa5504e34f>:0 
  at (wrapper remoting-invoke-with-check) System.Diagnostics.Process.Start()
  at System.Diagnostics.Process.Start (System.Diagnostics.ProcessStartInfo startInfo) [0x0001e] in <a85c1a570f9a4f9f9c3d2cfa5504e34f>:0 
  at System.Diagnostics.Process.Start (System.String fileName) [0x00006] in <a85c1a570f9a4f9f9c3d2cfa5504e34f>:0 
  at Pinta.Core.HelpActions.Bugs_Activated (System.Object sender, System.EventArgs e) [0x00001] in <c3b56c34d159455b913500476e0aff62>:0 
  at (wrapper managed-to-native) System.Reflection.RuntimeMethodInfo.InternalInvoke(System.Reflection.RuntimeMethodInfo,object,object[],System.Exception&)
  at System.Reflection.RuntimeMethodInfo.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x0007c] in <12b418a7818c4ca0893feeaaf67f1e7f>:0 
   --- End of inner exception stack trace ---
  at System.Reflection.RuntimeMethodInfo.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x0009a] in <12b418a7818c4ca0893feeaaf67f1e7f>:0 
  at System.Reflection.MethodBase.Invoke (System.Object obj, System.Object[] parameters) [0x00000] in <12b418a7818c4ca0893feeaaf67f1e7f>:0 
  at System.Delegate.DynamicInvokeImpl (System.Object[] args) [0x0010d] in <12b418a7818c4ca0893feeaaf67f1e7f>:0 
  at System.MulticastDelegate.DynamicInvokeImpl (System.Object[] args) [0x0000b] in <12b418a7818c4ca0893feeaaf67f1e7f>:0 
  at System.Delegate.DynamicInvoke (System.Object[] args) [0x00000] in <12b418a7818c4ca0893feeaaf67f1e7f>:0 
  at GLib.Signal.ClosureInvokedCB (System.Object o, GLib.ClosureInvokedArgs args) [0x0006d] in <918680a31aa84cb89cfa7cab56ea29b8>:0 
  at GLib.SignalClosure.Invoke (GLib.ClosureInvokedArgs args) [0x0000c] in <918680a31aa84cb89cfa7cab56ea29b8>:0 
  at GLib.SignalClosure.MarshalCallback (System.IntPtr raw_closure, System.IntPtr return_val, System.UInt32 n_param_vals, System.IntPtr param_values, System.IntPtr invocation_hint, System.IntPtr marshal_data) [0x0006c] in <918680a31aa84cb89cfa7cab56ea29b8>:0 
  ```

**Version**
I am using xubuntu 22.04

The version window says 1.6, your website says the newest one is 2.1, is this a problem for my distribution? If so I am sorry.

If I use `sudo apt upgrade pinta` apt says that pinta is already the newest version.

![image](https://github.com/PintaProject/Pinta/assets/5713211/067b45ff-667a-4051-ad67-c514073379b6)
![image](https://github.com/PintaProject/Pinta/assets/5713211/7c7c8e3c-9547-46c5-a222-e1a057520f36)

Thank you for writing this software otherwise. Works mostly fine, I will file more bugs if I notice more things :)

**Comment by cameronwhite** [comment]
Yeah that is a really old version of Pinta. I think this is the same as https://bugs.launchpad.net/pinta/+bug/1888883 which was fixed for Pinta 1.7, but I'd strongly suggest using Pinta 2.1 if possible (perhaps through the snap or flatpak package)

**Comment by BMaxV** [comment]
So, the right move would seem to be to use flatpak, but that doesn't respect the system theme I set up. I guess I will use the old version. Not your fault though.
